docs/.custom_wordlist.txt

```diff
@@ -0,0 +1,100 @@
+addons
+AlmaLinux
+API
+APIs
+apt's
+balancer
+BaseConfigurationError
+buildd
+BuilddBase
+BuilddBaseAliases
+CDN
+CentOS
+cfg
+changelog
+charmcraft
+Charmhub
+CLI
+config
+config’s
+config's
+devel
+diataxis
+Diátaxis
+dropdown
+EBS
+EKS
+favicon
+github
+Grafana
+hostname
+hostnames
+http
+https
+IAM
+implementers
+init
+installable
+Jira
+JSON
+Juju
+Kubeflow
+Kubernetes
+linter
+linux
+linuxcontainers
+lpcraft
+LXC
+lxd
+LXD
+LXDInstance
+LXDInstance’s
```

Comment:
We should probably just replace occurrences of [this right single quote character](https://www.fileformat.info/info/unicode/char/2019/index.htm) with regular apostrophes.